src/utils/formsg-utils.ts

```diff
@@ -7,13 +7,35 @@ export function getField(
 ): string | undefined {
   const response = responses.find(({ question }) => question === name)
 
-  return response?.answer
+  return response?.answer?.trim()
+}
+
+function trimAllStrings(
+  responseArray: string[] | string[][]
+): string[] | string[][] {
+  if (Array.isArray(responseArray)) {
+    responseArray.map((item) => {
+      if (Array.isArray(item)) {
+        return trimAllStrings(item)
+      }
+      if (typeof item === "string") {
+        return item.trim()
+      }
+      return item
+    })
+    return responseArray
+  }
+  return responseArray
 }
 
 export function getFieldsFromTable(
   responses: FormField[],
   name: string
 ): string[] | string[][] | undefined {
   const response = responses.find(({ question }) => question === name)
+  let answers = response?.answerArray
+  if (answers) {
+    answers = trimAllStrings(answers)
+  }
   return response?.answerArray
```

Comment:
Just checking - does this modify `response`? I think it might be better to rewrite it so it's clearly regardless

Reply:
nope, this is a bug, fixed here [08433bb](https://github.com/isomerpages/isomercms-backend/pull/698/commits/08433bb83269ceb74044f3e9e7a6dc6e1c4fd4e8)